apps/src/code-studio/pd/workshop_dashboard/components/survey_results/single_choice_responses.jsx

```diff
@@ -21,7 +22,16 @@ export default class SingleChoiceResponses extends React.Component {
         );
       }, 0);
     } else {
-      return Object.values(this.props.answers).reduce((sum, x) => sum + x, 0);
+      if (this.props.numRespondents !== undefined) {
+        // Multi-select questions will tell us how many respondents there were,
+        // so that we can correctly show the percentage of respondents who
+        // gave a certain answer.  (The default technique of counting the number
+        // of answers doesn't work when a single respondent can choose multiple
+        // answers, though it continues to work for single-select questions.)
+        return this.props.numRespondents;
```

Comment:
why don't we use the number of respondents for both single and multi select? that would make the code simpler and give us the same result.

Reply:
As discussed, there are still three codepaths in `summarize_workshop_surveys` on the server that return without adding the `num_respondents` field, and it's becoming a pretty big matrix to test, and so we're going to stick with this technique for now to remain backwards compatible with any server-side path that doesn't set this field.